javascript/index.js

```diff
@@ -1,40 +1,130 @@
-// This will print in the wrong order.
-// We added it as an example and to test that the arrays from data.js are loaded
-
-// 🚨🚨🚨 Comment out the below code before you start working on the code
-
-// Out of sync
-  getInstruction("mashedPotatoes", 0, (step1) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 1, (step2) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 2, (step3) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 3, (step4) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
-  }, (error) => console.log(error));
-  
-  getInstruction("mashedPotatoes", 4, (step5) => {
-    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
-    document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
-
-
+const potatoeImg = document.getElementById("mashedPotatoesImg");
+const steakImg = document.getElementById("steakImg");
+const broccoliImg = document.getElementById("broccoliImg");
+const brusselImg = document.getElementById("brusselsSproutsImg");
 
+const mashedPotatoesOl = document.getElementById("mashedPotatoes");
 // Iteration 1 - using callbacks
-// ...
+getInstruction("mashedPotatoes", 0, (step0) => {
```

Comment:
could you think of a way to get rid of the nesting? 
maybe something like a recursive function? i.e. a function that in the callback passed to getInstruction calls itself until some recursive depth is reached?